ajax/libs/zepto/package.json

```diff
@@ -20,6 +20,12 @@
       "files": [
         "**/*"
       ]
+    },
+    {
+      "basePath": "",
+      "files": [
+        "src/*"
+      ]
     }
   ]
-}
+}
```

Comment:
@sashberd Would you please try to use `./tools/fixFormat.js` under cdnjs folder to fix this problem?
Many thanks.

Reply:
I tried, nothing happened!

Reply:
@sashberd I tried `fixFormat.js` but the situation didn't happen, please try again, maybe you can just don't use the script.